modules/ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: PROCEDURE
+[id="ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries_{context}"]
+= Configuring the hub cluster to use unauthenticated registries
+
+You can configure the hub cluster to use unauthenticated registries.
+
+.Procedure
+
+. Edit the `AgentServiceConfig` CR to include the `unauthenticatedRegistries` parameter:
+
+[source,terminal]
+----
+$ apiVersion: agent-install.openshift.io/v1beta1
+kind: AgentServiceConfig
+metadata:
+  name: agent
+spec:
+  unauthenticatedRegistries:
+  - example.registry.com
+  - example.registry2.com
+  ...
+----
+
+Unauthenticated registries are listed under `spec.unauthenticatedRegistries` in the `AgentServiceConfig` resource.
+Any registry on this list is not required to have an entry in the pull secret validated by the `assisted-service`.
+`assisted-service` validates the [pull secret](hive-integration/kube-api-getting-started.md#2-create-a-pull-secret) provided for spoke cluster installation by making sure it contains the authentication information for every image registry used for installation.
+
+[NOTE]
+====
+[Mirror registries](#mirror-registry-configuration) will automatically be added to the ignore list and do not need to be added under `spec.unauthenticatedRegistries`.
+
+Specifying the `PUBLIC_CONTAINER_REGISTRIES` environment variable in the [ConfigMap override](#specifying-environmental-variables-via-configmap) is still supported and will completely overwrite the list to whatever is in the override.
```

Comment:
Likewise for these lines, the `[Mirror registries](#mirror-registry-configuration)` and `[ConfigMap override](#specifying-environmental-variables-via-configmap)` are leftover from a different document. Perhaps edit the linkage or remove the link altogether.

Reply:
Updated as suggested.